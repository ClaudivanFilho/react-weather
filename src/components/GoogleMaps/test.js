import React from 'react';
import {shallow} from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
import renderer from 'react-test-renderer';

import GoogleMaps from './index';

configure({ adapter: new Adapter() });

test('Test GoogleMaps Component', () => {
  const component = shallow(<GoogleMaps />);
  expect(component.text()).toEqual('<withScriptjs(withGoogleMap(Component)) />');
});

test('Test GoogleMaps Snapshot', () => {
  const component = renderer.create(
    <GoogleMaps />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})