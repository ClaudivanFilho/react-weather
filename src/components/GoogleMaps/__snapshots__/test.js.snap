// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test GoogleMaps Snapshot 1`] = `
<div
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
